Rlabkey/man/labkey.provenance.createProvenanceParams.Rd

```diff
@@ -9,7 +9,7 @@ Note: this function is in beta and not yet final, changes should be expected so
 labkey.provenance.createProvenanceParams(recordingId=NULL, name=NULL, description=NULL,
         runName=NULL, materialInputs=NULL, materialOutputs=NULL, dataInputs=NULL,
         dataOutputs=NULL, inputObjectUriProperty=NULL, outputObjectUriProperty=NULL,
-        objectInputs=NULL, objectOutputs=NULL, provenanceMap=NULL)
+        objectInputs=NULL, objectOutputs=NULL, provenanceMap=NULL, params=NULL, properties=NULL)
```

Comment:
I don't understand this CRAN check, but it looks like the TeamCity check is showing two NOTEs related to line widths for changes to this line and the example below at line 75:
https://teamcity.labkey.org/repository/download/LabKey_Publishing_Client_RApiBuild/1754404:id/build.tar.gz!/Rlabkey.Rcheck/00check.log